src/components/structures/LoggedInView.tsx

```diff
@@ -302,14 +305,27 @@ class LoggedInView extends React.Component<IProps, IState> {
         }
     };
 
+    _onHideToast = () => {
```

Comment:
This should be `private onHideToast` and using a less generic name - something like `onUsageLimitDismissed`